AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_LightAdded.py

```diff
@@ -68,13 +71,14 @@ def AtomEditorComponents_Light_AddedToEntity():
     2) Add Light component to the Light entity.
     3) UNDO the entity creation and component addition.
     4) REDO the entity creation and component addition.
-    5) Enter/Exit game mode.
-    6) Test IsHidden.
-    7) Test IsVisible.
-    8) Delete Light entity.
-    9) UNDO deletion.
-    10) REDO deletion.
-    11) Look for errors.
+    5) Set the light type to sphere.
```

Comment:
Is there a reason we're only testing sphere and not cycling through light types. Could be good for a p0 to make sure that all light types can be set.

Regardless, I'd leave this generic to "Set the Light type." that way in the future we can cycle through the light types.

Reply:
Updated to cycles through light types, and report result